repo/fsrepo/fsrepo.go

```diff
@@ -164,30 +180,17 @@ func Init(repoPath string, conf *config.Config) error {
 
 // Remove recursively removes the FSRepo at |path|.
 func Remove(repoPath string) error {
-	repoPath = path.Clean(repoPath)
```

Comment:
It went inline in the TildeExpansion call. I can yank it out to a separate line if you prefer that, but now all the path manipulation is very tightly grouped and I felt it reads well enough. Just a misleading diff.

```go
	expPath, err := u.TildeExpansion(path.Clean(repoPath))
	if err != nil {
		return nil, err
	}

	r := &FSRepo{
		path: expPath,
	}
```